anago

```diff
@@ -372,7 +383,7 @@ prepare_tree () {
 
   # rev base.go
   case $label in
-    beta*|official) rev_version_base $label ;;
+    beta*|rc|official) rev_version_base $label ;;
```

Comment:
`beta` has a weird case when a [three-point patch branch is created](https://github.com/ixdy/kubernetes-release/blob/e922b3d9cc780c736acecc3315fe456d59396e2c/lib/releaselib.sh#L341-L348).

(There's probably bug here if you try to create a 3-point branch when the release branch has an rc. I'm going to hope that doesn't happen.)